CMakeLists.txt

```diff
@@ -28,32 +28,32 @@ else()
 endif()
 
 #BOOST
-set(BOOST_ROOT /usr/)
 FIND_PACKAGE(Boost REQUIRED python)
 FIND_PACKAGE(PythonLibs REQUIRED)
+FIND_PACKAGE(OpenCV REQUIRED)
+
 include_directories(SYSTEM ${Boost_INCLUDE_DIR})
 include_directories(SYSTEM ${PYTHON_INCLUDE_DIR})
+include_directories(SYSTEM ${OpenCV_INCLUDE_DIR})
 
 include_directories(src)
-set(SRC_DIR src)
+set(SRC_DIR /set/path/pydarknet)
```

Comment:
If you do need to change this variable, better change it in a CMake's fasion, e.g. setting the variable in `cmake` command.